core/src/main/java/org/jdbi/v3/CollectorFactoryRegistry.java

```diff
@@ -29,7 +32,7 @@
  */
 class CollectorFactoryRegistry {
```

Comment:
So if we get rid of `collectInto` does all this `CollectorFactoryRegistry` stuff go away entirely?